<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigFile</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>465</width>
     <height>580</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>с</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> до</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время работы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество обрабатываемых сайтов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> %</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество повторных посещений</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время нахождения на странице</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество страниц перехода</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Переход по рекламным баннерам</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_adv">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> да</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Эмуляция ссылок поисковиков </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_search">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> да</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адреса сайтов для посещения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> мин.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> шт.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> шт.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> ч.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите параметры:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_transition_amount_to">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_time_on_website_to">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_repeatability">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_percentage">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_time_to">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_time_from">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_urls">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoNone</set>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID робота</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_robot_id">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_time_on_website_from">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> до</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>с</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> до</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>с</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_transition_amount_from">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
